Lib.AspNetCore.ServerSentEvents/IServerSentEventsService.cs

```diff
@@ -28,6 +28,14 @@ public interface IServerSentEventsService
         /// Gets the interval after which clients will attempt to reestablish failed connections.
         /// </summary>
         uint? ReconnectInterval { get; }
+
+        /// <summary>
+        /// Gets the callback that will be used to validate an incoming connection.
+        /// </summary>
+        /// <remarks>
+        /// If the callback handled the request, <c>false</c> is returned and the protocol will not be started.
+        /// </remarks>
+        Func<ServerSentEventsClientValidationArgs, Task<bool>> ValidationHandler { get; }
```

Comment:
I would suggest rephrasing naming and description around validator.

```
/// <summary>
/// Gets the validator which will be used for validating incoming connection.
/// </summary>
/// <remarks>
/// The validator should return true if the incoming connection is valid, otherwise false.
/// </remarks>
Func<ServerSentEventsConnectionValidationContext, Task<bool>> ConnectionValidator { get; }
```